skyportal/handlers/api/source.py

```diff
@@ -1756,6 +1777,13 @@ async def get(self, obj_id=None):
               type: boolean
             description: |
               If true, return only those objects which have not been labelled
+          - in: query
+            name: currentUserLabeller
+            nullable: true
+            schema:
+              type: boolean
+            description: |
+              If true and one of hasBeenLabeller or hasNotBeenLabelled is true, return only those objects which have not been labelled/not labelled by the current user. Otherwise, return results for all users.
```

Comment:
```suggestion
              If true and one of hasBeenLabeller or hasNotBeenLabelled is true, return only those objects which have been labelled/not labelled by the current user. Otherwise, return results for all users.
```

Reply:
Thanks!